README.md

```diff
@@ -2,38 +2,74 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-accounts-userservices/workflows/build/badge.svg)](https://github.com/cisagov/cool-accounts-userservices/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
-
-## Usage ##
-
-```hcl
-module "example" {
-  source = "github.com/cisagov/cool-accounts-userservices"
-
-  aws_region            = "us-west-1"
-  aws_availability_zone = "b"
-  subnet_id             = "subnet-0123456789abcdef0"
-
-  tags = {
-    Key1 = "Value1"
-    Key2 = "Value2"
-  }
-}
-```
-
-## Examples ##
-
-* [Deploying into the default VPC](https://github.com/cisagov/cool-accounts-userservices/tree/develop/examples/default_vpc)
+This project contains Terraform code to perform the initial configuration
+of a COOL User Services account. This Terraform code creates and configures
+the most basic resources needed to build out services and environments.
+
+It creates an IAM role that allows sufficient permissions to provision all
+AWS resources in this account. This role has a trust relationship with the
+COOL users account.
```

Comment:
Just a thought.

```suggestion
[COOL users account](https://github.com/cisagov/cool-accounts/tree/develop/users).
```

Reply:
It's a good idea @mcdonnnj - we reference many of the COOL accounts throughout our documentation and I think links like that could be helpful (at least the first time an account is mentioned in a document; every mention would probably be overkill).

However, I'm hesitant to do this right now because @jsf9k and I were recently discussing plans to split all of the subdirectories from [cisagov/cool-accounts](https://github.com/cisagov/cool-accounts) (e.g. `users`, `sharedservices`, `terraform`, etc.) into their own repositories (e.g. `cisagov/cool-accounts-users`, `cisagov/cool-accounts-sharedservices`, `cisagov/cool-accounts-terraform`, etc.), so all of these documentation URLs would need to be updated when that happens.

Reply:
I'm a big fan of breaking up [cisagov/cool-accounts](https://github.com/cisagov/cool-accounts), but once that's done I would like us to be more liberal about linking references like this one so it's easier to cross reference everything.

Reply:
Absolutely agree! 👍